src/options.rs

```diff
@@ -133,6 +133,15 @@ pub(crate) struct Config {
     #[structopt(short, long, help = "Base URL to check relative URls")]
     #[serde(default)]
     pub base_url: Option<String>,
+
+    #[structopt(
+        short,
```

Comment:
Personally I'd remove the `short` option in case we'd like to support an option where `-g` is fitting. Can't think of anything right now but I'd rather keep the set of options small for now and make it quite explicit that we're passing a github token here.
I'm fine either way, though. 😉

Reply:
Makes sense to me :+1: